apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,313 @@
+              />
+            </v-row>
+            <!-- Rule Body Info -->
+            <v-row dense>
+              <ChecklistRuleInfoBody :selected-rule="selectedRule" />
+            </v-row>
+            <!-- Rule Info Edit -->
+            <v-row dense>
+              <ChecklistRuleEdit
+                :selected-rule="selectedRule"
+                :sheet="sheet"
+                @enable-sheet="sheet = true"
+              />
+            </v-row>
+          </v-col>
+        </v-row>
+        <div class="text-center">
+          <v-bottom-sheet v-model="sheet" persistent inset>
+            <!-- Severity Override Justification -->
+            <ChecklistSeverityOverride
+              :selected-rule="selectedRule"
+              :sheet="sheet"
+              @disable-sheet="sheet = false"
+              @enable-sheet="sheet = true"
+            />
+          </v-bottom-sheet>
+        </div>
+      </v-container>
+    </template>
+  </Base>
+</template>
+
+<script lang="ts">
+import Base from './Base.vue';
+import Component from 'vue-class-component';
+import RouteMixin from '@/mixins/RouteMixin';
+import {SearchModule} from '@/store/search';
+import {
+  ChecklistFilter,
+  FilteredDataModule,
+  checklistRules
+} from '@/store/data_filters';
+import {
+  FileID,
+  SourcedContextualizedEvaluation,
+  SourcedContextualizedProfile
+} from '@/store/report_intake';
+import UploadButton from '@/components/generic/UploadButton.vue';
+import {ChecklistConverter, ChecklistVuln} from '@mitre/hdf-converters';
+import {InspecDataModule} from '@/store/data_store';
+import _ from 'lodash';
+import {saveSingleOrMultipleFiles} from '@/utilities/export_util';
+import IconLinkItem from '@/components/global/sidebaritems/IconLinkItem.vue';
+import {AppInfoModule} from '@/store/app_info';
+import ChecklistRulesTable from '@/components/global/checklist/ChecklistRulesTable.vue';
+import ChecklistRuleInfoHeader from '@/components/global/checklist/ChecklistRuleInfoHeader.vue';
+import ChecklistRuleInfoBody from '@/components/global/checklist/ChecklistRuleInfoBody.vue';
+import ChecklistRuleEdit from '@/components/global/checklist/ChecklistRuleEdit.vue';
+import ChecklistSeverityOverride from '@/components/global/checklist/ChecklistSeverityOverride.vue';
+import ExportButton from '@/components/generic/ExportButton.vue';
+
+@Component({
+  components: {
+    Base,
+    UploadButton,
+    IconLinkItem,
+    ChecklistRulesTable,
+    ChecklistRuleInfoHeader,
+    ChecklistRuleInfoBody,
+    ChecklistRuleEdit,
+    ChecklistSeverityOverride,
+    ExportButton
+  }
+})
+export default class Checklist extends RouteMixin {
+  /** Model for if all-filtered snackbar should be showing */
+  filterSnackbar = false;
+  controlSelection: string | null = null;
+
+  /** State variable to track status of bottom-sheet */
+  sheet = false;
+
+  /** State variable to track "Short ID" switch */
+  shortIdEnabled = true;
+
+  //** Variable for selected tab */
+  tab = null;
```

Comment:
delete unused variables such as this one

Reply:
Removed `tab` as it wasn't being used